sessions/react-component-testing/demo-end/components/Form/Form.test.js

```diff
@@ -0,0 +1,60 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import Form from ".";
+
+test("calls the onAddMovie handler with inputs on submit", async () => {
+  const user = userEvent.setup();
+  const handleAddMovie = jest.fn();
+  render(<Form onAddMovie={handleAddMovie} />);
+  const nameInput = screen.getByLabelText("Name");
+  const isLikedInput = screen.getByLabelText("Like");
+  const submitButton = screen.getByRole("button", { name: "Add" });
+  await user.type(nameInput, "The Matrix");
+  await user.click(isLikedInput);
+  await user.click(submitButton);
+  expect(handleAddMovie).toBeCalledWith({ name: "The Matrix", isLiked: true });
+});
+
+test("does not call the onAddMovie handler when the form is submitted without a name", async () => {
+  const user = userEvent.setup();
+  const handleAddMovie = jest.fn();
+  render(<Form onAddMovie={handleAddMovie} />);
+  const submitButton = screen.getByRole("button", { name: "Add" });
+  await user.click(submitButton);
+  expect(handleAddMovie).not.toBeCalled();
```

Comment:
`toBeCalled` is an alias. We should go with the default naming.
```suggestion
  expect(handleAddMovie).not.toHaveBeenCalled();
```